Cargo.toml

```diff
@@ -14,8 +14,6 @@ members = [
     "tools/check-guide",
     "tools/query-events",
 ]
-
-[patch.crates-io]
```

Comment:
Is this removed for a reason or is it just for aesthetic reasons ?

Reply:
Not sure how it got removed but there is no real reason to either leave it or keep it.

Reply:
Ok, with the commented block underneath I have a slight preference if it's kept then. Without the `[patch.crates-io]` the block might be confusing.
Or the commented block could also be removed if it seems better that way

Reply:
Oh right, didn't realize there was a comment block below, will re-add